src/main/resources/bouncer.yaml

```diff
@@ -1,17 +1,127 @@
+              schema:
+                type: string
+                example: The requested ID (ark:/21198/zz002dvwr6) was not found
+        '500':
+          description: There was an internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: The service failed to connect to its access control database
+  /cookie:
+    get:
+      summary: Get Authentication Cookie
+      description: The authentication cookie service
+      operationId: getCookie
+      parameters:
+      - in: query
+        name: origin
+        description: The origin of the page, consisting of a protocol, hostname and, optionally, port number
+        schema:
+          type: string
+        required: true
+      responses:
+        '200':
+          description: The cookie gets set in the response that includes Javascript to close the open tab/window
+          headers:
+            Set-Cookie:
+              description: An encrypted cookie value that contains the supplied origin and level of allowed access
+              schema:
+                type: string
+                example: iiifauth=asrvaaDe43AAqw34QpOoPJR4j5b1; Path=/; HttpOnly
+          content:
+            text/html:
+              schema:
+                type: string
+        '400':
+          description: The request didn't contain an origin or contained one that wasn't valid
+          content:
+            text/html:
+              schema:
+                type: string
+        '500':
+          description: There was an internal server error
+          content:
+            text/html:
+              schema:
+                type: string
+                example: The service failed to encrypt its cookie value
+  /token:
+    get:
+      summary: Get Authentication Token
+      description: The authentication token service
+      operationId: getToken
+      parameters:
+      - in: cookie
+        name: iiifauth
+        schema:
+          type: string
+          example: iiifauth=asrvaaDe43AAqw34QpOoPJR4j5b1; Path=/; HttpOnly
+        required: true
+      responses:
+        '200':
+          description: The access token response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  accessToken:
+                    type: string
+                  expiresIn:
+                    type: integer
+        '400':
+          description: The request didn't contain a valid cookie
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: The request did not contain a valid cookie
+        '500':
+          description: There was an internal server error
+          content:
+            text/plain:
+              schema:
+                type: string
+                example: The service failed to connect to its access control database
```

Comment:
Was this change intentional?

Reply:
(Just asking because the handlers are unimplemented.)

Reply:
Yes, this is IIIF-1117, which is just for the OpenAPI spec (none of the handlers are implemented yet; there are different tickets for those). I was trying to keep this in a separate PR but it seems to have slipped in here.